lib/screens/current_set.dart

```diff
@@ -105,14 +119,41 @@ class CurrentSetState extends State<CurrentSet> {
                           Text(
                             'December 12',
                             style: TextStyle(
-                                color: Colors.white, fontSize: bodyFont),
+                                color: Colors.white,
+                                fontWeight: FontWeight.bold,
+                                fontSize: bodyFont),
                           ),
                         ],
                       )
                     ],
                   ),
                 ),
               ],
+            )),
+        Container(
+            padding: const EdgeInsets.all(20.0),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                Text('Highest Rated',
+                    style: TextStyle(
+                      fontSize: headerFont,
+                    )),
+                Container(
+                    padding: const EdgeInsets.symmetric(vertical: 20.0),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        //todo: add route routing
+                        ratingWidget(
+                            function: () => print('top rope'),
+                            route: topRopeRoute),
+                        ratingWidget(
+                            function: () => print('bounder'),
```

Comment:
bounder?

Reply:
It's just a print statement to hold the palce until the actual function is put in. I don't think it matters

Reply:
PRINT THE BOUNDER!!!